src/Parser.mly

```diff
@@ -45,73 +49,73 @@
 typ:
     (* Unit type *)
   | UNIT
-    { Type.Unit }
+    { Location.mkLocByPosition Type.Unit $symbolstartpos $endpos}
```

Comment:
```suggestion
    { Location.locate Type.Unit $symbolstartpos $endpos }
```
Please keep the whitespace as in the original version.

Reply:
Done